action.yml

```diff
@@ -191,6 +200,44 @@ runs:
       set +x
     shell: bash
 
+  - name: Log the next step action
+    run: echo ▷ Figuring out change detection and coverage
+    shell: bash
+  - name: Determine change detection and coverage
+    id: compute-change-detection-coverage
+    run: |
+      # Compute whether to use change detection and coverage
+      import os
+      import pathlib
+      import sys
+
+      FILE_APPEND_MODE = 'a'
+      OUTPUTS_FILE_PATH = pathlib.Path(os.environ['GITHUB_OUTPUT'])
+
+      def set_output(name, value):
+          with OUTPUTS_FILE_PATH.open(FILE_APPEND_MODE) as outputs_file:
+              outputs_file.writelines(f'{name}={value}{os.linesep}')
+
+      # Input from GHA
+      pull_request_change_detection = '${{
```

Comment:
Let's apply `json.loads()` to this and make sure it's always a boolean in the Python-land.

Reply:
dfc469f410ea4bfe780341d1cd43fb2371f96f3e.